debian/patches/0060-add-simd-optimized-tonemapx-filter.patch

```diff
@@ -2714,331 +4163,1872 @@ Index: FFmpeg/libavfilter/vf_tonemapx.c
 +    { "param",        "tonemap parameter", OFFSET(param), AV_OPT_TYPE_DOUBLE, {.dbl = NAN}, DBL_MIN, DBL_MAX, FLAGS },
 +    { "desat",        "desaturation strength", OFFSET(desat), AV_OPT_TYPE_DOUBLE, {.dbl = 0}, 0, DBL_MAX, FLAGS },
 +    { "peak",         "signal peak override", OFFSET(peak), AV_OPT_TYPE_DOUBLE, {.dbl = 0}, 0, DBL_MAX, FLAGS },
++    { "apply_dovi",  "Apply Dolby Vision metadata if possible", OFFSET(apply_dovi), AV_OPT_TYPE_BOOL, { .i64 = 1 }, 0, 1, FLAGS },
```

Comment:
`range` option default to `-1`?

Reply:
```
[tonemapx @ 0000018213ccaa40] Unsupported output color range -1 ((null))
```
Oops. This needs to be handled internally.

Reply:
Handled by [6d0b354](https://github.com/jellyfin/jellyfin-ffmpeg/pull/447/commits/6d0b354ec5ad32772eea70bf9a4f6ca15610a91f)